src/ServiceControl.Monitoring/Retries/RetriesStore.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace ServiceControl.Monitoring
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Infrastructure;
+    using Timings;
+    public class RetriesStore : IntervalsStore, IProvideEndpointMonitoringData, IProvideEndpointInstanceMonitoringData
```

Comment:
I'm not a native English speaker but this name is somewhat strange `RetriesStore`. Shouldn't it be `RetryStore `?